test/fixtures.js

```diff
@@ -365,3 +366,35 @@ export const createGraphDataResponse = (overrides = {}) => ({
   },
   ...overrides,
 })
+export const createAssessmentDayDataMetadata = (overrides = {}) => ({
+  path: '',
+  filetype: '',
+  encoding: '',
+  basename: ' ',
+  dirname: ' ',
+  mtime: 0,
+  size: 0,
+  uid: 0,
+  gid: 0,
+  mode: 0,
+  role: ' ',
+  study: ' ',
+  participant: ' ',
+  assessment: ' ',
+  units: ' ',
+  start: ' ',
+  end: ' ',
+  extension: ' ',
+  time_end: ' ',
+  Consent: ' ',
+  Active: 1,
+  ...overrides,
+})
+
+export const createAsessmentVariable = (overrides = {}) => ({
+  name: '',
+  assessment: '',
+  ...overrides,
+})
+export const createAssessmentVariables = (overrides = []) => [...overrides]
+export const createParticipantAssessments = (overrides = []) => [...overrides]
```

Comment:
I think we just need the singular versions of these to create the objects. The tests can construct the arrays of the data using the fixture functions.